openff/bespokefit/executor/services/coordinator/stages.py

```diff
@@ -530,6 +533,20 @@ async def _inject_bespoke_qc_data(
 
             target.reference_data = local_qc_data
 
+            from qcelemental.molutil import guess_connectivity
+
+            # Check that connectivity still matches the input molecule
+            original_mol: Molecule = Molecule.from_smiles(input_schema.smiles)
```

Comment:
For the case where the fragmentation stage produces different molecules we will need to use them as the reference as this is currently compared to the input molecule.  This causes an error to be raised even though the calculation runs fine. Confirmed to break with `C1=CC=C(C=C1)C2=CC=CC(=C2)C#N`.

Reply:
Ohhhh yeah good point. OK.